src/content/en/2020/accessibility.md

```diff
@@ -8,17 +8,442 @@ authors: [oluoluoxenfree, alextait1]
+
+<figure markdown>
+| Role         | Desktop | Mobile |
+| ------------ | ------- | ------ |
+| button       | 25.20%  | 24.51% |
+| navigation   | 22.97%  | 21.76% |
+| dialog       | 19.01%  | 18.22% |
+| search       | 17.94%  | 17.59% |
+| presentation | 17.83%  | 16.31% |
+
+  <figcaption>
+    {{ figure_link(
+      caption="Top 5 ARIA roles on the web.",
+      sheets_gid="TODO",
+      sql_file="TODO.sql"
+    ) }}
+  </figcaption>
+</figure>
+ 
+#### ARIA Attributes
+
+ARIA attributes can be assigned to HTML elements to enhance the accessibility of the interface. Respecting the first rule of ARIA, they should not be used to achieve something that can be done with native HTML.
+
+##### Labeling and describing elements with ARIA
+
+The browser’s accessibility tree has a computation system that assigns the accessible name (if there is one) to a control, widget, group, or landmark such that it can be announced by assistive technology. There is a specificity ranking that happens to determine which value is assigned to the accessible name. 
+
+The accessible name can be derived from an element’s content (such as button text), an attribute (such as an image alt text value), or an associated element (such as a programmatically associated label for a form control. For more information about accessible names see Léonie Watson’s article, [What is an accessible name?](https://developer.paciellogroup.com/blog/2017/04/what-is-an-accessible-name/)
+
+We can also use ARIA to provide accessible names for elements. There are 2 ARIA attributes that accomplish this, [aria-label](https://www.w3.org/WAI/GL/wiki/Using_aria-label_to_provide_labels_for_objects), [aria-labelledby](https://www.w3.org/WAI/GL/wiki/Using_aria-labelledby_to_provide_a_name_for_user_interface_controls) . Either of these attributes will “win” the accessible name computation and override the natively derived accessible name, so use them with caution and be sure to test with a screen reader or look at the accessibility tree to confirm that the accessible name is what was expected. When using ARIA to name an element, it is important to ensure that the [WCAG 2.5.3, Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html) criterion has not been violated, which expects visible labels to be at least a part of its accessible name. 
+
+
+The `aria-label` element allows a developer to provide a string value and this will be used for the accessible name for the element. We found that 40.44% of desktop pages and 38.72% of mobile home pages had at least one element with the aria-label attribute, making it the most popular ARIA attribute for providing accessible names. 
```

Comment:
```suggestion

The `aria-label` element allows a developer to provide a string value and this will be used for the accessible name for the element. We found that 40.44% of desktop pages and 38.72% of mobile home pages had at least one element with the `aria-label` attribute, making it the most popular ARIA attribute for providing accessible names. 
```